helm-chart/values.yaml

```
@@ -10,12 +10,13 @@ acendTraining:
         tag: ""
         tagsuffix: ""
       ingress:
-        annotations: 
-          kubernetes.io/ingress.class: nginx-public
-        useDefaultSecret: true
+        labels:
+          public: "true"
+        useDefaultSecret: "true"
```

These should be booleans:

```suggestion
        useDefaultSecret: true
```

---

applied